R/multitables.R

```diff
@@ -108,3 +108,27 @@ newMultitable <- function (formula, data, name, ...) {
     u <- crPOST(shojiURL(data, "catalogs", "multitables"), body=toJSON(payload))
     invisible(Multitable(crGET(u)))
 }
+
+#' Import a Multitable
+#'
+#' Allows you to import a multitable from a different dataset
+#' @param data an object of class \code{CrunchDataset} in which to create the
+#' multitable
+#' @param multitable an object of class \code{Multitable} that you want copied
+#' to the new dataset
+#' @param ... Additional multitable attributes to set. Options include
+#' \code{is_public}.
```

Comment:
TBH I have to google "roxygen markdown" to find their new vignette that would answer that question every time i have to do anything more than the two things I just named.

Another complication to your question is that `Multitable` is probably both a class name and a constructor function.

I also don't know that the class/constructor documentation is useful enough to link to right now. It's not expected that users will init Multitable objects; rather, they'll create them with `newMultitable/importMultitable` or load them by extracting from the multitable catalog.